.github/workflows/discord-bot-deployment.yml

```diff
@@ -17,3 +17,30 @@ jobs:
         run: cd discord-bot && npm ci
       - name: Deploy Serverless Bot on AWS
         run: cd discord-bot && npm run deploy
+        env:
+          # Discord Bot Token, Guild ID and Client ID
+          TOKEN: ${{ secrets.TOKEN }}
+          PUBLIC_KEY: ${{ secrets.PUBLIC_KEY }}
+          CLIENT_ID: ${{ secrets.CLIENT_ID }}
+          GUILD_ID: ${{ secrets.GUILD_ID }}
+
+          # Faucet Contract Address and Minter Private Key
+          CONTRACT_ADDRESS: '0x872f3534a525023F9E42Ad69b788B44fc958D5aa'
+          PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+
+          # Subspace EVM-Domain Token Symbol, RPC Endpoint and Blockscout Explorer URL
+          TOKEN_SYMBOL: 'tSSC'
+          RPC_ENDPOINT: 'https://nova.gemini-3g.subspace.network/ws'
+          EXPLORER_URL: 'https://nova.subspace.network'
+
+          # Debug Mode (true/false)
+          DEBUG_MODE: 'true'
+
+          # Slack integration
+          SLACK_ENABLED: 'true'
```

Comment:
Let's please put this inside the env as well, just to ensure it's easier to control in case anything goes wrong